tiled/server/app.py

```diff
@@ -474,6 +475,12 @@ def override_get_settings():
                 settings.database_settings.uri = (
                     settings.database_settings.uri or "sqlite://"
                 )
+        if authenticators:
+            # ProxiedOIDCAuthenticator cannot be used alongside other authentication providers
+            if len(authenticators) == 1:
+                authenticator = next(iter(authenticators.values()))
+                if isinstance(authenticator, ProxiedOIDCAuthenticator):
+                    settings.authenticator = authenticator
```

Comment:
This passes silently if `ProxiedOIDCAuthenticator` is used alongside other authentication providers. It should raise a clear exception here, I think.

As an aside, it may be that Tiled should drop support for more using than one authenticator in the future, at which point this could be simplified.